Index: settings.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>pluginManagement {\r\n    repositories {\r\n        gradlePluginPortal()\r\n        google()\r\n        mavenCentral()\r\n    }\r\n}\r\ndependencyResolutionManagement {\r\n    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)\r\n    repositories {\r\n        google()\r\n        mavenCentral()\r\n    }\r\n}\r\nrootProject.name = \"Jobs\"\r\ninclude ':app'\r\ninclude ':data-user'\r\ninclude ':data-jobs'\r\ninclude ':ui-user'\r\ninclude ':ui-jobs'\r\ninclude ':domain-jobs'\r\ninclude ':domain-user'\r\ninclude ':builder'\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settings.gradle b/settings.gradle
--- a/settings.gradle	(revision 0754c58ca48ce2df2b650f5f2ee14c08adec7444)
+++ b/settings.gradle	(date 1672558468245)
@@ -1,8 +1,8 @@
 pluginManagement {
     repositories {
+        mavenCentral()
         gradlePluginPortal()
         google()
-        mavenCentral()
     }
 }
 dependencyResolutionManagement {
Index: build.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>buildscript {\r\n    repositories {\r\n        mavenCentral()\r\n        google()\r\n    }\r\n    dependencies {\r\n        classpath(\"org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31\")\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle.kts b/build.gradle.kts
--- a/build.gradle.kts	(revision 0754c58ca48ce2df2b650f5f2ee14c08adec7444)
+++ b/build.gradle.kts	(date 1672558507656)
@@ -1,9 +1,18 @@
+plugins {
+
+    id("com.github.ben-manes.versions").version("0.41.0")
+    id("nl.littlerobots.version-catalog-update").version("0.7.0")
+}
+
 buildscript {
+
+
     repositories {
         mavenCentral()
         google()
     }
+
     dependencies {
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31")
     }
-}
\ No newline at end of file
+}
Index: gradle/wrapper/gradle-wrapper.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#Wed Dec 14 11:17:04 IRST 2022\r\ndistributionBase=GRADLE_USER_HOME\r\ndistributionUrl=https\\://services.gradle.org/distributions/gradle-7.3.3-bin.zip\r\ndistributionPath=wrapper/dists\r\nzipStorePath=wrapper/dists\r\nzipStoreBase=GRADLE_USER_HOME\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle/wrapper/gradle-wrapper.properties b/gradle/wrapper/gradle-wrapper.properties
--- a/gradle/wrapper/gradle-wrapper.properties	(revision 0754c58ca48ce2df2b650f5f2ee14c08adec7444)
+++ b/gradle/wrapper/gradle-wrapper.properties	(date 1672559481167)
@@ -1,6 +1,6 @@
 #Wed Dec 14 11:17:04 IRST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
 distributionPath=wrapper/dists
 zipStorePath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
